### PR TITLE
`mekotronics`: updated DT+LCD driver from blueberry_edge_patch_230617 from vendor

### PR DESCRIPTION
**Pull request by rpardini**
- meko: updated DT's from `blueberry_edge_patch_230617` from vendor 
- meko: remove silly indentation changes; bring back amazingfate's hdmiin-sound changes in the v10 
- meko: vk2c21 LCD driver for the R58X-PRO 
- meko: add new DTs to Makefile 
- meko: es8311: v10, preserve "arm64: dts: rockchip: rk3588: Use mclkout_i2sx" from 005807f 
- meko: v12 MeiZhuo: drop DP changes from vendor

### CODE REVIEW

File: arch/arm64/boot/dts/rockchip/rk3588-blueberry-edge-v10.dtsi [hunk 1]
```diff
@@ -397,9 +398,9 @@
 		adc-volume = <0xbf>;    /* 0dB */
 		dac-volume = <0xbf>;    /* 0dB */
 		aec-mode = "adc left, adc right";
-		clocks = <&mclkout_i2s0>;
+		clocks = <&cru I2S0_8CH_MCLKOUT>;
```
Comment (amazingfate) [inline arch/arm64/boot/dts/rockchip/rk3588-blueberry-edge-v10.dtsi]: Does this change make audio works?

Comment (amazingfate) [reply arch/arm64/boot/dts/rockchip/rk3588-blueberry-edge-v10.dtsi]: Take a look at this commit: https://github.com/armbian/linux-rockchip/commit/005807f776ef16a62e58b32cb7db9991d6b189f4

**Review by amazingfate — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/rockchip/rk3588-blueberry-edge-v10.dtsi (hunk 1 above)
Comment (rpardini) [reply arch/arm64/boot/dts/rockchip/rk3588-blueberry-edge-v10.dtsi]: Super thanks, I had missed it, I will preserve your changes in the next push.

Comment (rpardini) [reply arch/arm64/boot/dts/rockchip/rk3588-blueberry-edge-v10.dtsi]: Interesting that the `v12` from vendor already had those. I re-picked 005807f for the `v10`

**Review by amazingfate — APPROVED** [review]

**Pull request merged**